doc-content/enterprise-only/openshift/environment-authoring-con.adoc

```diff
@@ -1,16 +1,43 @@
-[id='environment-authoring-con']
+[id='environment-authoring-con'] 
+ifdef::PAM[]
 = Authoring environment
 You can deploy an environment for creating and modifying processes using {CENTRAL}. It consists of {CENTRAL} for the authoring work and {KIE_SERVER} for test execution of the processes.
+endif::PAM[]
+ifdef::DM[]
+= Authoring or managed server environment
+You can deploy an environment for creating and modifying services using {CENTRAL} and for running them in {KIE_SERVERS} managed by {CENTRAL}. This environment consists of {CENTRAL} and one or more {KIE_SERVERS}.
 
-Depending on your needs, you can deploy either a single authoring environment or a high-availability (HA) authoring environment.
+You can use {CENTRAL} both to develop services and to deploy them to one or several {KIE_SERVERS}. For example, you can deploy test versions of services to one {KIE_SERVER} and production versions to another {KIE_SERVER}.
```

Comment:
I think that this line is not needed. I am actually more confused. If am I understand this line, it basically says that we can connect more KIE Servers to one Central, right?

Reply:
Yes, but we gotta offer a use case for that...